### DISCUSSION TITLE
I did a :PackerSync by mistake

### DISCUSSION THREAD

**Discussion by nappalm** [wbthomason/packer.nvim#1068] [General]
By mistake the packages were updated, and now everything starts to fail 😖
What can I do to shuffle my versions back to how they were?

**Comment by bangedorrunt** [comment]
Unless you did a snapshot and use PackerSnapshotRollback :(

**Reply by nappalm** [reply]
Yes, I have a snapshot, I already did what you mention `:PackerSnapshotRollback packer_snapshot` but nothing happens.

**Reply by bangedorrunt** [reply]
Hmm, how about remove packer folder, reinstall, then restore snapshot. This is only solution I could think of :(

**Reply by nappalm** [reply]
I'm new to this, where should that folder be?

**Reply by bangedorrunt** [reply]
It depends on your platform, generally it should be here
<img width="452" alt="Screen Shot 2022-09-13 at 6 42 20 pm" src="https://user-images.githubusercontent.com/1514823/189854978-1179d93a-913d-4659-885a-df0f082742d8.png">

**Reply by bangedorrunt** [reply]
And your rollback file should be located here
<img width="316" alt="Screen Shot 2022-09-13 at 6 45 26 pm" src="https://user-images.githubusercontent.com/1514823/189855737-2370e8da-37bc-4fd3-8768-f6c321094338.png">
In above example, my snapshot named `foo`

**Reply by bangedorrunt** [reply]
So, tried to remove `packer` folder, reinstall, then `PackerSnapshotRollback foo` to see if it fix your issue.
